guide.md

```diff
@@ -0,0 +1,18 @@
+[Back](./README.md)
+
+# Guide
+
+1. The list is sorted with alphabetical sections. With exceptions for "Numbers", "Symbols" and "Others" which are special.
+2. In section, abbrs are sorted in alphabetical order.
+3. Abbrs are in word-abbr format.
+4. Colored circles before the abbr mean the degree of recommendation:
+
+-  🟢 Recommended
+-  🟡 Context-sensitive
+-  🔴 Not recommended
+
+Unrecommended abbrs can still be considered.
+The most important thing is to remain consistent in your code.
+
+4. Context abbrs are more suitable in certain cases. The context is specified between { } after the abbr.
+5. If a word has more possible abbrs they are sorted from the most recommended to the least one. If there's more of one kind it's sorted in alphabetical order.
```

Comment:
Each word should have only one recommended abbreviation.

Reply:
Okk.